code/lib/cli/rendererAssets/svelte/Button.svelte

```diff
@@ -41,3 +40,36 @@
 >
   {label}
 </button>
+
+<style>
+  .storybook-button {
```

Comment:
the formatting on this line does not match the others

Reply:
fixed, I added indentation to all other lines :)